### PR TITLE
compute & readme updates

### PR DESCRIPTION
**Pull request by skeppeler**
### Issue

#501 

### Description

```
commit 4d75e94e2da9f8354974f81cc1437e308f23f92e (HEAD -> chore/compute, origin/chore/compute)
Author: Steve Keppeler <keppeler@amazon.com>
Date:   Tue Mar 12 14:58:48 2024 -0400

    chore: update documentation on role and model access

    * update documentation to specify the developer role may be used
    * add an issue as a hint to the user to validate models are accessible

commit dfbece5c086c02f86d62b8d712153614759b5939
Author: Steve Keppeler <keppeler@amazon.com>
Date:   Tue Mar 12 14:43:06 2024 -0400

    chore: increase disk space for CDK deploy
```

### Testing

How was this change tested?

### Additional context

Add any other context about the PR here.

---

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by alexforsyth — APPROVED** [review]

File: packages/blueprints/gen-ai-chatbot/src/blueprint.ts [hunk 1]
```diff
@@ -208,6 +214,17 @@ export class Blueprint extends ParentBlueprint {
       content: 'Log into your chatbot and use the bot console on the left to add custom data to the bot.',
     });
 
+    {
+      const models = Object.values(requiredModels);
+      const last = models.pop();
+      const modelNames = `${models.join(', ')}, and ${last}`;
+      new Issue(this, 'validate-model-access', {
```
Comment (alexforsyth) [inline packages/blueprints/gen-ai-chatbot/src/blueprint.ts:221]: Is there a way we can do this check with a model invoke in the workflow?

**Review by alexforsyth — APPROVED** [review]

File: packages/blueprints/gen-ai-chatbot/src/blueprint.ts (hunk 1 above)
Comment (alexforsyth) [reply packages/blueprints/gen-ai-chatbot/src/blueprint.ts:221]: Feel free to merge and address in a follow up if you want

Comment (skeppeler) [reply packages/blueprints/gen-ai-chatbot/src/blueprint.ts:221]: yup, was going to address that in a separate PR. a little more involved so wanted to get this out first.

Comment (Lock128) [reply packages/blueprints/gen-ai-chatbot/src/blueprint.ts:221]: x

**Review by Lock128 — COMMENTED** [review]
see comments

File: packages/blueprints/gen-ai-chatbot/src/workflows.ts [hunk 2]
```diff
@@ -87,6 +87,7 @@ export const getDeploymentWorkflow = (blueprint: ParentBlueprint, options: Optio
       dependsOn: [...bootstrapActions, 'BuildFrontend'],
       computeName: {
         Type: 'EC2',
+        Fleet: 'Linux.Arm64.2XLarge',
```
[the remark this answers is not in the document]
Comment (skeppeler) [reply packages/blueprints/gen-ai-chatbot/src/workflows.ts:90]: I believe you're correct - the workflow documentation [does not specify this](https://docs.aws.amazon.com/codecatalyst/latest/userguide/compute.html#compute.on-demand) but that does seem to be the case in my testing. I will roll this back.

**Pull request merged**